tests/random_lax_test.py

```diff
@@ -1341,7 +1341,12 @@ def test_binomial_dtypes(self):
     key = jax.random.key(0)
     n = jax.numpy.float16(100)
     p = jax.numpy.float16(0.5)
-    jax.random.binomial(key, n, p)  # doesn't error
```

Comment:
This doesn't seem like the appropriate test to modify, as this one is a regression test for a specific issue.

Better would be to use both `jtu.dtypes.floating + jtu.dtypes.integer` here:  https://github.com/jax-ml/jax/blob/6259cfe31742e8985f9522beb1d64d7903002950/tests/random_lax_test.py#L1302